### ISSUE TITLE
Change Thermal Tuning to slow down the rate that body temperature changes, rather than completely neutralizing it

### ISSUE THREAD

**Issue by Forstride** [Glitchfiend/ToughAsNails#832]
Instead of completely neutralizing body temperature, the Thermal Tuning enchantment should just slow down the rate at which body temperature changes, but ONLY when it's moving towards an extreme value (So that trying to cool yourself down in a hot biome doesn't take longer for example).

**Comment by sirrobzeroone** [comment]
Just wanted to add a +1 to this suggestion, liekwise findin the full neutralizing of temperature when using 1 armour piece with Thermal Tuning a tad powerful. 

Could it be graded similar to leaf and Wool, so that only a full suit of armour with all pieces with Thermal Tuning enchantment then move towards full negating or the larger pieces providing more weighting ie chest plate, legs, helmet and finally boots adding slightly different weightings of effectivness to negate temperature effects?

really like the mod it adds a really nice element to minecraft to have to worry about etting to hot or cold

**Comment by dan-w1231** [comment]
Agree. I assumed what you described is what it did but after finding the enchant and applying it to chest, it's the equivalent of disabling temperature in the config. 

If someone doesn't want temperature they can just disable it in the config. Even the strongest worn item shouldn't entirely negate temperature, or it should come at great cost or difficulty, in keeping with the mod.
